presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -345,38 +376,83 @@ private void print(int indent, String format, Object... args)
         output.append(indentString(indent)).append(value).append('\n');
     }
 
-    private void printStats(int indent, PlanNodeId planNodeId)
+    private void print(int indent, String format, List<Object> args)
+    {
+        print(indent, format, args.toArray(new Object[args.size()]));
+    }
+
+    private void printStats(int intent, PlanNodeId planNodeId)
+    {
+        printStats(intent, planNodeId, false, false);
+    }
+
+    private void printStats(int indent, PlanNodeId planNodeId, boolean printInput, boolean printFiltered)
     {
         if (!stats.isPresent()) {
             return;
         }
+
         long totalMillis = stats.get().values().stream()
                 .mapToLong(node -> node.getWallTime().toMillis())
                 .sum();
-        PlanNodeStats stats = this.stats.get().get(planNodeId);
-        if (stats == null) {
-            output.append(indentString(indent))
-                    .append("Cost: unknown, Output: unknown \n");
+
+        PlanNodeStats nodeStats = stats.get().get(planNodeId);
+        if (nodeStats == null) {
+            output.append(indentString(indent));
+            output.append("Cost: ?");
```

Comment:
I think `?` will make output more compact. Take a look at example:
`PartitionedOutputOperator := Collisions avg.: ? (?% est.), Collisions std.dev.: ?%`.
If we put `unknown` in place of `?` then the line would get really long and won't be great for readability.